pkg/addon/versioned_registry.go

```diff
@@ -26,8 +26,8 @@ import (
 	"github.com/pkg/errors"
 	"helm.sh/helm/v3/pkg/chart/loader"
 	"helm.sh/helm/v3/pkg/repo"
+	"k8s.io/klog"
```

Comment:
This is not aligned, we are using both klog v1 and v2

Reply:
good catch